torch/distributed/_composable/fsdp/_fsdp_state.py

```diff
@@ -223,6 +235,7 @@ def _root_post_backward_final_callback(self) -> None:
             if self._state_ctx.is_last_backward:
                 self._comm_ctx.post_forward_order.clear()
             self._state_ctx.post_backward_final_callback_queued = False
+            logger.debug("FSDP::root_post_backward")
```

Comment:
Should this be moved up to beginning of method?

Reply:
moved

Reply:
i still see `logger.debug("FSDP::root_post_backward")` at the end. is it moved?